src/GoogleFitOnFhir/Clients/GoogleFit/GoogleFitDataImporterContext.cs

```diff
@@ -0,0 +1,15 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace GoogleFitOnFhir.Clients.GoogleFit
+{
+    public class GoogleFitDataImporterContext
```

Comment:
I think we are safe to bump the GoogleFitDatasetRequestLimit value up to 100. We should log the EventDataBatch size and number of events when we call SendAsync, so we can get a sense of the average size of the response.